source/Octopus.Tentacle/Security/AesEncryption.cs

```diff
@@ -74,7 +74,11 @@ Aes GetCryptoProvider(byte[]? iv = null)
 
         static byte[] GetEncryptionKey(string encryptionPassword)
         {
+#if NET8_0_OR_GREATER
+            using var passwordGenerator = new Rfc2898DeriveBytes(encryptionPassword, PasswordPaddingSalt, PasswordSaltIterations, HashAlgorithmName.SHA1);
```

Comment:
NET8 requires the encryption algorithm is explicit as the other overload has been marked as obsolete.

Internally, it used the SHA1 algorithm.